tezos/new_context/src/working_tree/storage.rs

```diff
@@ -0,0 +1,607 @@
+// Copyright (c) SimpleStaking, Viable Systems and Tezedge Contributors
+// SPDX-License-Identifier: MIT
+
+use std::{
+    cmp::Ordering,
+    convert::{TryFrom, TryInto},
+    mem::size_of,
+};
+
+use modular_bitfield::prelude::*;
+use static_assertions::assert_eq_size;
+use tezos_timing::StorageMemoryUsage;
+
+use crate::kv_store::entries::Entries;
+
+use super::{
+    string_interner::{StringId, StringInterner},
+    Node,
+};
+
+#[bitfield]
+#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, PartialOrd, Ord)]
+pub struct TreeStorageId {
+    /// Note: Must fit in NodeInner.entry_id (61 bits)
+    #[skip]
+    __: B14,
```

Comment:
@sebastiencs what is this? can you add a comment? or give it a meaningful name, prefixed with `__` to make it explicit that it is not used (at least that is how I am interpreting it now)